From 25eb76b3d2726f2f1de759901971ae53b2539dc4 Mon Sep 17 00:00:00 2001
From: Harshula Jayasuriya <harshula@gmail.com>
Date: Mon, 21 Mar 2011 14:49:37 +0000
Subject: Add #define XK_SINHALA

Add #define XK_SINHALA so that the Sinhala keysyms can be used by
the lk xkb keymap.

Signed-off-by: Harshula Jayasuriya <harshula@gmail.com>
Reviewed-by: Daniel Stone <daniel@fooishbar.org>
---
diff --git a/src/KeyBind.c b/src/KeyBind.c
index ac25ce2..221cedd 100644
--- a/src/KeyBind.c
+++ b/src/KeyBind.c
@@ -44,6 +44,7 @@ in this Software without prior written authorization from The Open Group.
 #define XK_CAUCASUS
 #define XK_VIETNAMESE
 #define XK_XKB_KEYS
+#define XK_SINHALA
 #include <X11/keysymdef.h>
 #include <stdio.h>
 
--
cgit v0.8.3-6-g21f6
